compiler/resolution/callDestructors.cpp

```diff
@@ -1700,6 +1700,31 @@ static void checkForInvalidGlobalUses() {
 *                                                                             *
 ************************************** | *************************************/
 
+static bool isActualCopyInitialized(CallExpr* call, ArgSymbol* formal) {
```

Comment:
it would be good to put a comment describing the purpose of this and the ast pattern it looks for